src/http/controller.hpp

```diff
@@ -63,12 +67,37 @@ class DedeController : public oatpp::web::server::api::ApiController
            REQUEST(std::shared_ptr<IncomingRequest>, req),
            QUERIES(QueryParams, queryParams))
   {
-    // TODO(sileht): why do serialize the query string to char*
-    // to later get again a APIData...
+    auto jsonObjectMapper
```

Comment:
You may create this object-mapper on the stack:

```cpp
oatpp::parser::json::mapping::ObjectMapper jsonObjectMapper;
```

Or, if the controller has it's own mapper - use that one:

```cpp
getDefaultObjectMapper();
```